spec/models/audit/event/authn/login_spec.rb

```diff
@@ -55,7 +60,11 @@
     it 'uses the WARNING log level' do
       expect(subject.severity).to eq(Syslog::LOG_WARNING)
     end
-
+    it 'contains the not-found user field' do
+      expect(subject.structured_data).to match(hash_including({
+          Audit::SDID::AUTH => { authenticator: authenticator_name, service: "rspec:webservice:my-service", user: "not-found" }
```

Comment:
see if you can reuse the not-found constant and if not please add it as constant in this file
Do it for all test files